uint8       priority
uint8       socket_id
uint8[3]    payload